playgrounds/vue/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Question.

Why do we've two ```tsconfig``` files here?

Reply:
Vue CLI has created them. As I see, one extends `@vue/tsconfig/tsconfig.node.json` and the other - `@vue/tsconfig/tsconfig.web.json`

I think it's better to leave it as is for our dev-ops guys.